File: 2015/gara1/figonacci/att/figonacci.pas
function enumera(N, M: longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    enumera := 42;
end;


var
    N, M   : longint;
    fr, fw : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N, M);

    writeln(fw, enumera(N, M));
    close(fr);
    close(fw);
end.

File: ed1/gara1/figonacci/att/figonacci.c
#include <stdio.h>
#include <assert.h>

int enumera(int N, int M) {
    // Mettete qui il codice della soluzione
    return 42;
}


int main() {
    FILE *fr, *fw;
    int N, M;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(2 == fscanf(fr, "%d%d", &N, &M));

    fprintf(fw, "%d\n", enumera(N, M));
    fclose(fr);
    fclose(fw);
    return 0;
}
